src/jit/codegenarmarch.cpp

```diff
@@ -2717,8 +2717,6 @@ void CodeGen::genJmpMethod(GenTree* jmp)
         // In case of a jmp call to a vararg method ensure only integer registers are passed.
         if (compiler->info.compIsVarArgs)
         {
-            assert((genRegMask(argReg) & RBM_ARG_REGS) != RBM_NONE);
```

Comment:
Why was it incorrect?
Is `// In case of a jmp call to a vararg method ensure only integer registers are passed.` a valid comment now?

Reply:
The assert and the comment conflict with each other.

Reply:
The comment is correct

Reply:
I do not understand, `RBM_ARG_REGS` contains all int argument registers (`(RBM_ARG_0|RBM_ARG_1|RBM_ARG_2|RBM_ARG_3|RBM_ARG_4|RBM_ARG_5|RBM_ARG_6|RBM_ARG_7)`), the opposite define for floats is `RBM_FLTARG_REGS`.

The comment says that all `argReg` has to be int, that means `(genRegMask(argReg) & RBM_ARG_REGS) != RBM_NONE`. Where am I wrong?

Reply:
Sorry the issue is that R8 can also be defined. Will change the assert.

Reply:
Could you please move the comment above the assert inside the `if`?

Reply:
Done